__tests__/app.test.js

```diff
@@ -77,16 +77,23 @@ describe('app', () => {
                         expect(reviews).toBeSortedBy('created_at', { descending: true })
                     })
                 })
-                test.only('status 200: allows reviews to be sorted by category, asc by default', () => {
+                test('status 200: allows reviews to be sorted by category, asc by default', () => {
```

Comment:
Sorting by category would mean you want results from a certain category. For example: '?category=social deduction'